### ISSUE TITLE
`Try code` on the docs pages

### ISSUE THREAD

**Issue by jbampton** [One-Language/Document#107]
I really like how `Lucee` has done their docs with the ability to run the code on the same page

https://docs.lucee.org/reference/functions/abs.html

Not sure what you were thinking for `run.onelang.org` ??

Crystal gives you a page to play in too

https://play.crystal-lang.org/#/cr

**Comment by BaseMax** [comment]
Good idea, we can rename run to play. but I think run is a good word too. What is your comment?

**Comment by BaseMax** [comment]
Good idea john. I think if we create the next generation of onelang website. we can design this feature too

**Comment by jbampton** [comment]
They have this run box on some pages:

![Screen Shot 2022-08-28 at 6 47 00 pm](https://user-images.githubusercontent.com/418747/187065717-ba84ff47-ba30-4f99-b4ac-fc9666bae8d2.png)

https://docs.lucee.org/reference/functions/abs.html

**Comment by jbampton** [comment]
Golang do it well too with the `Go Tour`

https://go.dev/tour/basics/1
